### DISCUSSION TITLE
Edit current cell in $EDITOR or $VISUAL

### DISCUSSION THREAD

**Discussion by LiamKearn** [saulpw/visidata#1490] [Q&A] [answered]
Just asking if this is already present before I start learning about commands and plugins :)

Would be really nice to be able to use `e` for quick edits and `E` to jump into vi or emacs for proper editing.

Another TUI application that does this really nicely is lazygit.

I'm so excited for using this project. Hoping to never have to do another `cat xyz.csv | column -ts',' | less -RS`!

**Comment by saulpw** [comment, marked as answer]
Hi @LiamKearn, you can `e` to edit and then `Ctrl+O` to jump into $EDITOR.

**Reply by LiamKearn** [reply]
Thanks Saul,

Exactly what I was after. You've made one hell of a usable TUI. It didn't take long to learn the basics :)
